modules/apps/portal-vulcan/portal-vulcan-test/src/testIntegration/java/com/liferay/portal/vulcan/internal/graphql/servlet/test/BaseGraphQLServlet.java

```diff
@@ -98,6 +98,33 @@ public BaseGraphQLServlet.TestDTO createTestDTO(
 
 	}
 
+	public static class TestPagination {
```

Comment:
I think this class is very confusing to understand what is the test about.

It seems you are getting a Pagination element, as in Vulcan there is already a Pagination, but that is not the purpose.

I think it would be better to use a Page<> and then you create a Page of a TestDTO to simulate the real use case that we are dealing with. What do you think?

Reply:
Yeah, I think that I could take another check in order to unify with the TestDTO